src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements DrawAble {
```

Comment:
```suggestion
public class Circle extends Figure{
```
якщо батьківський клас імплементує інтерфейс - це автоматично переходить і до всіх наслідників